.github/workflows/build-macos-bypass.yaml

```diff
@@ -40,7 +29,7 @@ on:
 jobs:
   build:
     name: Build on Mac OS
```

Comment:
Can we update this?

Reply:
No, the name has to be the same as the real workflow to pass required checks and get PRs added to the merge queue.

Reply:
I thought the workflow name just needs to be the same (see line 11). Does the job name have to be the same as well?

Reply:
Yes, all job names need to be the same.